internal/builder/lifecycle.go

```diff
@@ -120,3 +120,8 @@ func (l *lifecycle) binaries() []string {
 	}
 	return binaries
 }
+
+// SupportedLinuxArchitecture returns true for each binary architecture available at https://github.com/buildpacks/lifecycle/releases/
+func SupportedLinuxArchitecture(arch string) bool {
+	return arch == "arm64" || arch == "ppc64le" || arch == "s390x"
```

Comment:
Is it not possible to get this directly from the lifecycle image instead of hard-coding it? Also, maybe we just try to download whatever architecture the user provided and fail if it is not available? It would be annoying to have to keep updating this list when new arch support is added.

Reply:
We can create an issue for it, we are doing this for `arm64`  on pack 0.33.0.
I added `ppc64le` and `s390x`, but I agree we can do it better

Reply:
Let's make an issue for it (see https://github.com/buildpacks/pack/pull/2086#discussion_r1610085824)